infrastructure-formula/infrastructure/salt/files/etc/systemd/system/salt-stage.service

```diff
@@ -0,0 +1,27 @@
+# Service unit to validate and copy a Salt Git repository.
+# Copyright (C) 2024 Georg Pfuetzenreuter <georg.pfuetzenreuter@suse.com>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+{{ pillar.get('managed_by_salt_formula', '# Managed by the infrastructure formula') }}
+{%- set pistis = salt['pillar.get']('infrastructure:salt:pistis', {}) %}
+
+[Unit]
+Description=Salt repository validation
+
+[Service]
+Type=oneshot
+EnvironmentFile=/etc/pistis
+ExecStart=pistis{{ ' -gitlab ' ~ pistis.gitlab if 'gitlab' in pistis }} -repository {{ directory }}{{ ' -ignore-merge' if pistis.get('ignore-merge') is True }}{{ ' -loglevel ' ~ pistis.loglevel if 'loglevel' in pistis }}
```

Comment:
Where does pistis come from? I guess a pkg.installed in pistis.sls might be useful ;-)

Reply:
Right, how could I forget

Reply:
Added via https://github.com/openSUSE/salt-formulas/compare/92825d556d55785aa687fea93c0ee55487ceb81d..41c8733ab4025e0945c6f786f61ce4c1580a1ee3, thanks.